Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -435,6 +437,64 @@ int CLuaEngineDefs::EngineRestoreModel ( lua_State* luaVM )
     return 1;
 }
 
+int CLuaEngineDefs::EngineRequestModel ( lua_State* luaVM )
+{
+    SString strModelType;
+
+    CScriptArgReader argStream ( luaVM );
+    argStream.ReadString ( strModelType );
+
+    if ( !argStream.HasErrors () )
+    {
+        eClientModelType eModelType;
+        if ( stricmp( strModelType, "ped" ) == 0 )
```

Comment:
``strModelType == "ped"`` would be better here. No real need to make it case-insensitive or use C functions.